packages/starksheet-cairo/pyproject.toml

```diff
@@ -3,12 +3,13 @@ name = "onsheet"
 version = "0.1.0"
 description = ""
 authors = ["Clément Walter <clement0walter@gmail.com>"]
-packages = [{ include = "*.py" }]
+packages = [{ include = "**/*.py" }]
 
 [tool.poetry.dependencies]
 python = "~3.9"
 cairo-lang = "^0.11"
 openzeppelin-cairo-contracts = "^0.4.0"
+frozendict = "^2.3.8"
```

Comment:
there is a frozendict dep requiring explicitly 2.3.6, however this specific version is not found with pip (there is a 2.3.5 and a 2.3.7 but no 2.3.6). My guess would be this specific version has been invalidated. Not an expert of python and pip so I did not dig too deep into this

Reply:
not sure what happened to you but there is a frozendict 2.3.6 available on pypi https://pypi.org/project/frozendict/2.3.6/ and I was actually able to delete and reinstall my venv, so I'd rather not add this here plz